.travis.yml

```diff
@@ -59,7 +59,7 @@ addons:
 env:
   global:
     ## The Travis CI environment allows us two cores, so let's use both.
-    - MAKEFLAGS="-j 2"
+    - MAKEFLAGS="-j 2 V=0"
```

Comment:
I'm not sure this is the best strategy. libtool seems to be echoing the entire compiler command line with each compilation wrapper invocation, which seems to nearly double the size of the build log. We want the entire compiler command line in the log file because that can be important for troubleshooting a failed build. Is there some way to get libtool to be quieter? Also the `--disable-silent-rules` is what turns on the verbosity in the first place, so maybe that's the better place to make this sort of change?

Reply:
@tlyu:
> Is there some way to get libtool to be quieter?

Yes, [we can use the LIBTOOL environment variable](https://www.gnu.org/software/automake/manual/html_node/Libtool-Flags.html).

(It's _also_ possible to use `AM_LIBTOOLFLAGS`. The question is whether we want to disable libtool verbosity in Travis only or always.)

> Also the --disable-silent-rules is what turns on the verbosity in the first place, so maybe that's the better place to make this sort of change?

The `V=0` and `--enable-silent-rules` [seem to do the same job](https://www.gnu.org/software/automake/manual/html_node/Automake-Silent-Rules.html#Automake-Silent-Rules). The difference is whether one wants to do that at `./configure` time or at `make` invocation time.

Thanks for the other comments in the Track ticket! I'll reply to them soon.